README.md

```diff
@@ -36,6 +36,7 @@ sudo make install
 - pkgutils (CRUX)
 - emerge (Gentoo)
 - dnf (Fedora)
+- rpm-ostree (Fedora Silverblue)
```

Comment:
Please keep this list in alphabetical order.

Reply:
Fixed